test/CSAccounting.t.sol

```diff
@@ -546,9 +546,9 @@ abstract contract CSAccountingBondStateBaseTest is
         accounting.setBondCurve(0, curveId);
     }
 
-    function _lock(uint256 id, uint256 amount) internal virtual {
+    function _lock(uint256 amount) internal virtual {
```

Comment:
I think explicit is better than implicit

Reply:
all the other similar funcs doesn't have such ability, and there were no tests using this argument with other values than 0
so I just made it aligned with other helpers